ec-service/src/main/java/org/dcsa/endorsementchain/service/EblEnvelopeService.java

```diff
@@ -144,12 +146,17 @@ EblEnvelope signedEblEnvelopeToEblEnvelope(
       String transportDocumentJson,
       String platformHost) {
 
-    TransportDocument transportDocument =
-      TransportDocument.builder()
+    TransportDocument transportDocument = transportDocumentRepository.findById(eblEnvelopeTO.documentHash())
+      .map(td -> {
+        td.reimported();
+        return td;
```

Comment:
I don't think the return is needed here

Reply:
It is as `reimported` is a `void`.  But I can use the same trick you did in `export()` where it just `return this;` at the end to avoid the return here.

Reply:
i'm also fine with this